common/membership/grpc_resolver.go

```diff
@@ -61,7 +61,7 @@ func initializeBuilder(monitor Monitor) GRPCResolver {
 	return GRPCResolver{}
 }
 
-func (g *GRPCResolver) MakeURL(service primitives.ServiceName) string {
+func MakeResolverURL(service primitives.ServiceName) string {
```

Comment:
nit: Don't love that this is in `grpc_resolver.go` now that it also applies to HTTP.

Reply:
Cool, I can move this to another file.